packages/tab/src/react/__stories__/index.story.js

```diff
@@ -1,11 +1,9 @@
 import addons from '@storybook/addons'
-import core from '@pluralsight/ps-design-system-core'
 import React from 'react'
 import { storiesOf } from '@storybook/react'
-import Theme from '@pluralsight/ps-design-system-theme/react'
 import themeDecorator from '@pluralsight/ps-design-system-storybook-addon-theme'
 
-import Tab from '../react'
+import Tab from '../index.js'
```

Comment:
you could just do `..` but it's all the same thing in the end

Reply:
Yeah, we should chat about this at some point.  It's been a style thing for me for a long time to do exactly as you say (".."), but there are new developments about browser native modules and node modules that make me wonder if we should start doing "../index.js".